scripts/vm_run_tests.psm1

```diff
@@ -567,62 +349,24 @@ function Invoke-ConnectRedirectTestsOnVM
     $UdpProxyParameters = "--protocol udp --local-port $ProxyPort"
 
     $ParamaterArray = @($TcpServerParameters, $TcpProxyParameters, $UdpServerParameters, $UdpProxyParameters)
-    $VMArray = @($VMName)
-
-    Add-FirewallRuleOnVM -VM $VMName -RuleName "Redirect_Test" -ProgramName $ProgramName -LogFileName $LogFileName
+    Add-FirewallRule -RuleName "Redirect_Test" -ProgramName $ProgramName -LogFileName $LogFileName
 
     # Start TCP and UDP listeners on both the VMs.
-    foreach ($vm in $VMArray)
+    foreach ($param in $ParamaterArray)
```

Comment:
```suggestion
    foreach ($parameter in $ParameterArray)
```
fix typo ("Paramater" -> "Parameter") and avoid unnecessary abbreviations ("param") per coding conventions in this project